src/bsp/bootloader/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(BSP_FILES
         ${CMAKE_CURRENT_SOURCE_DIR}/Core/Src/stm32u5xx_hal_timebase_tim.c
         ${CMAKE_CURRENT_SOURCE_DIR}/Core/Src/stm32u5xx_it.c
         # ${CMAKE_CURRENT_SOURCE_DIR}/Core/Src/syscalls.c
-        ${CMAKE_CURRENT_SOURCE_DIR}/Core/Src/sysmem.c
+        # ${CMAKE_CURRENT_SOURCE_DIR}/Core/Src/sysmem.c
```

Comment:
Interestingly, the new compiler failed on a duplicate definition of `_sbrk` in the bootloader.